components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/cmd/FunctionCmd.java

```diff
@@ -0,0 +1,100 @@
+package org.wso2.apimgt.gateway.cli.cmd;
+
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.Parameters;
+import org.apache.commons.io.FileUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.wso2.apimgt.gateway.cli.exception.CLIRuntimeException;
+import org.wso2.apimgt.gateway.cli.utils.GatewayCmdUtils;
+import org.wso2.apimgt.gateway.cli.utils.RouteUtils;
+
+import java.io.File;
+import java.io.PrintStream;
+import java.util.List;
+
+@Parameters(commandNames = "function", commandDescription = "add function")
+public class FunctionCmd implements GatewayLauncherCmd {
+
+    private static final Logger logger = LoggerFactory.getLogger(FunctionCmd.class);
```

Comment:
Shall we change te static constants to uppercase?
```suggestion
    private static final Logger LOGGER = LoggerFactory.getLogger(FunctionCmd.class);
```

Reply:
changes are added